bin/common.ml

```diff
@@ -19,6 +19,7 @@ type t =
   { debug_dep_path        : bool
   ; debug_findlib         : bool
   ; debug_backtraces      : bool
+  ; debug_extra_backtraces      : bool
```

Comment:
This new feature doesn't seem to be used in this PR. Could you extract it in a separate PR? I also feel like it needs a convincing use case.

Reply:
Right. Sorry, I wanted to use this twice so far, but both use cases eventually disappeared. I'll keep it in a branch for now and wait until another use case pops up. 

The intended use case is to store backtraces on allocation of certain object (e.g. Memo.lazy_) to know where it came from.